prompt/prompt-executor/prompt-executor-clients/prompt-executor-google-client/src/commonMain/kotlin/ai/koog/prompt/executor/clients/google/GoogleLLMClient.kt

```diff
@@ -357,7 +357,7 @@ public open class GoogleLLMClient(
             responseJsonSchema = responseFormat?.responseJsonSchema,
             temperature = if (model.capabilities.contains(LLMCapability.Temperature)) prompt.params.temperature else null,
             candidateCount = if (model.capabilities.contains(LLMCapability.MultipleChoices)) prompt.params.numberOfChoices else null,
-            maxOutputTokens = 2048,
+            maxOutputTokens = prompt.params.maxTokens ?: model.maxOutputTokens?.toInt() ?: 2048,
```

Comment:
`GoogleGenerationConfig.maxOutputTokens` is optional. So to me the value from prompt should simply be coerced to the value of the model.

Something like:
```kotlin
maxOutputTokens =
  model.maxOutputTokens?.let { max ->
    prompt.params.maxTokens.coerceIn(0, max)
  } ?: prompt.params.maxTokens
```

Reply:
I think the exercise can be as trivial as `maxOutputTokens = prompt.params.maxTokens`. I updated the PR.